src/dafny/Math.md

```diff
@@ -0,0 +1,8 @@
+
+## The `Math` module {#sec-math}
+
+The `Math` module currently contains a few integer-related math functions (e.g. Min, Max, Abs).
+- Functions over integers using multiply, divide and modulo are handled in the
+`NonlinearArith` module, along with lemmas about those functions.
+- Integer-related functions over collections of integers (e.g. minimum of a sequence) are
+available in the `Collections` module.
```

Comment:
Good callout - feels weird, but perhaps #92 will put this in a more logical place.

Reply:
It made sense to have the properties like minimum of a sequence in the Seq module -- the various lemmas that were needed to prove the functionality were there anyway.